### PR TITLE
Adjust documentation of `get_block_file_template()`

### PR DESCRIPTION
**Pull request by felipeelia**
This PR adjusts `get_block_file_template()` documentation per [55929](https://core.trac.wordpress.org/ticket/55929)

Trac ticket: https://core.trac.wordpress.org/ticket/55929

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/block-template-utils.php [hunk 1]
```diff
@@ -811,7 +811,9 @@ function get_block_template( $id, $template_type = 'wp_template' ) {
 }
 
 /**
- * Retrieves a single unified template object using its id.
+ * Retrieves a unified template object based on a theme file.
+ *
+ * This is a fallback of {@see get_block_template()}, used when no templates are found in the database.
```
Comment (peterwilsoncc) [inline src/wp-includes/block-template-utils.php]:
```suggestion
 * This is a fallback of get_block_template(), used when no templates are found in the database.
```

The docs parser will pick this up and link without the need for a `@see` annotation.

**Review by peterwilsoncc — COMMENTED** [review]
Just a minor issue with the docblock standards, otherwise this looks good.

File: src/wp-includes/block-template-utils.php (hunk 1 above)
Comment (felipeelia) [reply src/wp-includes/block-template-utils.php]: Awesome, @peterwilsoncc, good to know. I've changed that in 368d397 now. Thanks!

**Review by peterwilsoncc — APPROVED** [review]

**Comment by peterwilsoncc** [comment]
Merged in https://core.trac.wordpress.org/changeset/54380 / https://github.com/WordPress/wordpress-develop/commit/2b2f1fd885a6bf948ef5ca83229951f59ceea99d

**Pull request closed without merging**